tutorials/intro-express/1-welcome.qmd

```diff
@@ -0,0 +1,310 @@
+
+1. Take the above code and save it to a file. Here we named it `app.py`
+2. Click on the play button (red circle in the image below)j
+
+You will see the terminal run the `shiny run` command for you automatically.
+The output will look something like this
+
+
+```bash
+$ python -m shiny run --port 55901 --reload --autoreload-port 55902 app-010-simple.py
+INFO:     Will watch for changes in these directories: ['~/Desktop/py-shiny-example']
+INFO:     Uvicorn running on http://127.0.0.1:55901 (Press CTRL+C to quit)
+INFO:     Started reloader process [24969] using WatchFiles
+INFO:     Started server process [24986]
+INFO:     Waiting for application startup.
+INFO:     Application startup complete.
+INFO:     127.0.0.1:56426 - "GET /?vscodeBrowserReqId=1737097751843 HTTP/1.1" 200 OK
+```
+
+This will run the application on port `55901` and automatically reload and update
+as you make changes to the `app.py` file.
+
+1. You will see the app build in the bottom terminal and open in the viewer on the side
+2. Move the slider and see how the output reacts
+3. Congratulations, you made your first shiny for python application!
+
+![](img/010-run_app.png)
+
+::: {.callout-tip}
+## Naming your files
+
+If you start your file with the word `app`,
+the shiny for python extension will recognize
+it as an application and you will be able to see the "play" button to run your application.
+You can also name your file `app-getting_started.py`
+and you will still get the shiny extension play button.
+
+To have Shiny for Python work well with the VS Code extensions and for you to go through
+the next series of lessons.
+We recommend either one of the following file naming conventions:
+
+1. Create separate folders for each app example you will create and save separate `app.py` files in each folder
+2. Create separate `app*.py` files in the same directory (e.g., `app-01.py`, `app-02.py`)
+:::
+
+:::
+
+## Run your shiny application
+
+In addition to the play button in Positron, you can manually run your application from
+the command line.
+This is useful if you wish to specify your own port or want to rename your application
+without the `app` prefix.
+
+```bash
+shiny run my_app.py
+```
+
+::: {.callout-note}
+If you named your application `app.py` you can omit it form the command and only use `shiny run`.
+The `app.py` is the default file shiny looks for to run in the current directory.
+Otherwise, you can pass in the name of the file that you wish to run.
+The `app` prefix used in the example above is used to signal the VS Code shiny extension
+to display the run app button.
+:::
+
+:::{.callout-tip}
+## Helpful run options
+
+Some useful options you can pass the `shiny run` command are:
+
+- `--port`: pass in a custom port, e.g., `--port 8000`.
+  This will run the app on the specified port,
+  instead of a random port.
+  This makes it easier to have the same browser window open as you stop and start your application.
+- `--reload`: Enables auto-reload
```

Comment:
I think you should include `--reload` when you show the first `shiny run` command